infra/cifuzz/docker.py

```diff
@@ -36,3 +51,30 @@ def delete_images(images):
   command = ['docker', 'rmi', '-f'] + images
   utils.execute(command)
   utils.execute(['docker', 'builder', 'prune', '-f'])
+
+
+def get_base_docker_run_args(out_dir, sanitizer='address', language='c++'):
+  """Returns arguments that should be passed to every invocation of 'docker
+  run'."""
+  docker_args = _DEFAULT_DOCKER_RUN_ARGS[:]
+  docker_args += [
+      '-e',
+      f'SANITIZER={sanitizer}',
+      '-e',
+      f'FUZZING_LANGUAGE={language}',
+  ]
+  docker_container = utils.get_container_name()
+  if docker_container:
+    docker_args += ['--volumes-from', docker_container, '-e', 'OUT=' + out_dir]
+  else:
+    docker_args += ['-v', f'{out_dir}:/out']
+  return docker_args, docker_container
+
+
+def get_base_docker_run_command(out_dir, sanitizer='address', language='c++'):
+  """Returns part of the command that should be used everytime 'docker run' is
+  invoked."""
+  docker_args, docker_container = get_base_docker_run_args(
+      out_dir, sanitizer, language)
+  command = _DEFAULT_DOCKER_RUN_COMMAND[:] + docker_args
```

Comment:
just use .copy() here instead of []. same above.

Reply:
Done.